http/swagger.yml

```diff
@@ -6929,4 +6933,35 @@ components:
         flux:
           description: The Flux script to run for this task.
           type: string
-      required: [flux]
\ No newline at end of file
+        token:
+          description: The token to use for authenticating this task when it executes queries. If omitted, uses the token associated with the request that creates the task.
+          type: string
+      required: [flux]
+    TaskUpdateRequest:
+      type: object
+      properties:
+        status:
+          description: Starting state of the task. 'inactive' tasks are not run until they are updated to 'active'
+          default: active
+          type: string
+          enum:
+            - active
+            - inactive
+        flux:
+          description: The Flux script to run for this task.
+          type: string
+        name:
+          description: Override the 'name' option in the flux script.
+          type: string
+        every:
+          description: Override the 'every' option in the flux script.
+          type: string
+        cron:
+          description: Override the 'cron' option in the flux script.
+          type: string
+        offset:
+          description: Override the 'offset' option in the flux script.
+          type: string
+        token:
+          description: Override the existing token associated with the task.
+          type: string
```

Comment:
add carriage return to end of file.

Reply:
I'm not going to wait for another CI cycle to get the EOL there, when it wasn't there before. Do we need to standardize on [EditorConfig](https://editorconfig.org/) or something else to ensure the style going forward?